bitnami/postgresql/templates/_helpers.tpl

```diff
@@ -412,6 +412,6 @@ Return the path to the CA cert file.
 {{- if .Values.tls.autoGenerated -}}
     {{- printf "%s-crt" (include "common.names.fullname" .) -}}
 {{- else -}}
-    {{ required "A secret containing TLS certificates is required when TLS is enabled" .Values.tls.certificatesSecret }}
+    {{ required "A secret containing TLS certificates is required when TLS is enabled" (tpl .Values.tls.certificatesSecret .) }}
```

Comment:
Just to avoid undesirable effects of `tpl` WDYT?

```suggestion
    {{ printf "%s" ( tpl ( required "A secret containing TLS certificates is required when TLS is enabled" .Values.tls.certificatesSecret ) . )  }}
```